### ISSUE TITLE
[2.0] Conjunction with components: click events fired twice on mobile devices

### ISSUE THREAD

**Issue by frederikbosch** [runspired/ember-mobiletouch#67]
Inside components we have something like.

```js
var $next = this.$('.next'); 
$next.click(function (eventClick) {
  eventClick.preventDefault();
  // do something
});
```

But these click events fire twice (sometimes) when ember-mobiletouch is installed and while using it on a mobile device (desktop is fine). Am I missing something from the docs? Is my threshold too low? We are on Ember 1.11.1 with ember-mobiletouch 2.0.0-beta.1. Below is the mobile config.

```
ENV.mobileTouch = {
    tune: {
      tap: {time: 250, threshold: 9},
      press: {time: 500, threshold: 9},
      swipe: {direction: 7, velocity: 0.5, threshold: 9},
      pan: {direction: 6},
      pinch: {},
      rotate: {}
    }
  };
```

**Comment by runspired** [comment]
@frederikbosch can you try this with 1.4.11

**Comment by frederikbosch** [comment]
@runspired That solved it!

**Comment by frederikbosch** [comment]
Renamed it, to be closed when is not encountered any more in 2.0. Thank you for your quick help.

**Comment by runspired** [comment]
Closing as this direction for 2.0 was abandoned.
